README.md

```diff
@@ -7,6 +7,11 @@
 
 This repository is the source for the Docker Hub image `containerbase/buildpack`. Commits to `main` branch are automatically built and published.
 
+## Apt proxy
+
+You can pass a custom temporary Apt proxy at build or runtime when installing new packages via `APT_HTTP_PROXY` arg.
```

Comment:
Do we support any runtime apt use? Assuming the user doesn't run as root

Reply:
Only if the user is root. 😉